int32 xmin
int32 xmax
int32 ymin
int32 ymax

float64 probability

int64 id
string label
